src/styles/getTooltipStyles.js

```diff
@@ -0,0 +1,117 @@
+import {Dimensions} from 'react-native';
+import spacing from './utilities/spacing';
+import styles from './styles';
+import themeColors from './themes/default';
+import fontFamily from './fontFamily';
+import variables from './variables';
+
+const GUTTER_WIDTH = 16;
```

Comment:
**Idea for a comment:** Explain what is the gutter width used for and why is it `16`

Reply:
Should it be a constant in `variables`?

Reply:
I'm not sure `GUTTER_WIDTH` will be relevant anywhere else, so I think it should be left here. I'll add a comment tho